src/test/java/bookstore/CategoryControllerTest.java

```diff
@@ -0,0 +1,177 @@
+package bookstore;
```

Comment:
```suggestion
package bookstore.controller;
```
Use the same package structure